app/services/supported_remediations.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'yaml'
+
+# Service for Supported Remediations
+class SupportedRemediations
+  class << self
+    def revision
+      raw_supported_remediations['revision']
+    end
+
+    private
+
+    def clear
+      @raw_supported_remediations = nil
+      @ansible_task_checksum = SsgConfigDownloader.ssg_ansible_tasks_checksum
+    end
+
+    def ssg_ansible_tasks_changed?
```

Comment:
As I said earlier, here you don't really need caching as this is a one-time running task in a cronjob. Its pod gets deleted after each run, therefore, any cache will be invalidated...

Reply:
Right, sorry I should've moved this PR to draft, I'm trying to figure out why the PR check keeps failing. It's timing out even when I set up the SupportedRemediations `revision` method like this:
```
def revision
      YAML.safe_load(SsgConfigDownloader.ssg_ansible_tasks)
 end
```

It seems like this should be pretty fast though so I'm not sure why it's timing out?